<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow_EZ_Tools</class>
 <widget class="QMainWindow" name="MainWindow_EZ_Tools">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>640</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>480</width>
    <height>640</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>481</width>
    <height>640</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Arial</family>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>EZ_Tools</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget_Tabs">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>482</width>
      <height>640</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>1024</width>
      <height>1024</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <property name="movable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="tab_EZ_Shading">
     <property name="palette">
      <palette>
       <active>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>92</red>
           <green>61</green>
           <blue>100</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Highlight">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>121</green>
           <blue>242</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Link">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>234</red>
           <green>156</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </active>
       <inactive>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>92</red>
           <green>61</green>
           <blue>100</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Highlight">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>255</red>
           <green>121</green>
           <blue>242</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Link">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>234</red>
           <green>156</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </inactive>
       <disabled>
        <colorrole role="Button">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>92</red>
           <green>61</green>
           <blue>100</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Highlight">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>51</red>
           <green>153</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
        <colorrole role="Link">
         <brush brushstyle="SolidPattern">
          <color alpha="255">
           <red>234</red>
           <green>156</green>
           <blue>255</blue>
          </color>
         </brush>
        </colorrole>
       </disabled>
      </palette>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{ color: white }
QPushButton{ background-color: rgb(92, 61, 100)}
QPushButton:enabled { color: white } 
QPushButton:hover:!pressed { color: rgb(42, 245, 72) }
QPushButton:hover:!pressed { background-color:rgb(170, 113, 185) }
QPushButton:pressed { color: black }
QPushButton:pressed {background: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(228, 228, 228, 255), stop:0.0852273 rgba(241, 187, 255, 255), stop:0.431818 rgba(179, 118, 194, 255), stop:1 rgba(92, 61, 100, 255));
        padding-top: 2px;
        padding-left: 3px;}</string>
     </property>
     <attribute name="title">
      <string>EZ_Shading</string>
     </attribute>
     <widget class="QLabel" name="label_mainImage">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>10</y>
        <width>480</width>
        <height>101</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QWidget" name="widget_EZ_Shading_001" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>150</y>
        <width>471</width>
        <height>61</height>
       </rect>
      </property>
      <widget class="QLabel" name="label_refFile">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>60</width>
         <height>23</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>Open Reference File</string>
       </property>
       <property name="text">
        <string>REF. File   :</string>
       </property>
      </widget>
      <widget class="QComboBox" name="comboBox_types_list">
       <property name="geometry">
        <rect>
         <x>68</x>
         <y>0</y>
         <width>61</width>
         <height>23</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>Props</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Characters</string>
        </property>
       </item>
      </widget>
      <widget class="QPushButton" name="pushButton_open_001">
       <property name="geometry">
        <rect>
         <x>421</x>
         <y>0</y>
         <width>51</width>
         <height>23</height>
        </rect>
       </property>
       <property name="whatsThis">
        <string/>
       </property>
       <property name="text">
        <string>Open</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_open_002">
       <property name="geometry">
        <rect>
         <x>421</x>
         <y>30</y>
         <width>51</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Open</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_dash">
       <property name="geometry">
        <rect>
         <x>68</x>
         <y>30</y>
         <width>349</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>---------------------------------------------------------------------------------------</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_mayaFile">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>0</y>
         <width>60</width>
         <height>23</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>Open Shader File</string>
       </property>
       <property name="text">
        <string>Maya File :</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_dash_2">
       <property name="geometry">
        <rect>
         <x>136</x>
         <y>0</y>
         <width>281</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>---------------------------------------------------------------------------------------</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="widget_EZ_Shading_002" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>210</y>
        <width>471</width>
        <height>281</height>
       </rect>
      </property>
      <widget class="QPushButton" name="pushButton_stepSix">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>190</y>
         <width>453</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Create Sky</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_stepSeven">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>220</y>
         <width>453</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Add Subdivision (Select Group)</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_stepTwo">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>453</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string> Load ' KAZ_W ' Preset</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_stepOne">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>453</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Switch 2 V-Ray</string>
       </property>
       <property name="autoRepeat">
        <bool>true</bool>
       </property>
      </widget>
      <widget class="QSlider" name="horizontalSlider_Y_lightRotation">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>160</y>
         <width>428</width>
         <height>22</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>Change the Light Direction</string>
       </property>
       <property name="maximum">
        <number>360</number>
       </property>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
      <widget class="QLabel" name="label_lightRotationNumber">
       <property name="geometry">
        <rect>
         <x>430</x>
         <y>210</y>
         <width>31</width>
         <height>21</height>
        </rect>
       </property>
       <property name="layoutDirection">
        <enum>Qt::LeftToRight</enum>
       </property>
       <property name="text">
        <string>0</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_stepFive">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>130</y>
         <width>453</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Create Light</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_stepFour">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>100</y>
         <width>453</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Create Cam</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_stepThree">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>453</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Texture Gamma Correct</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="widget_widget_EZ_Shading_003" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>490</y>
        <width>471</width>
        <height>41</height>
       </rect>
      </property>
      <widget class="QPushButton" name="pushButton_ImageSaveFile">
       <property name="geometry">
        <rect>
         <x>410</x>
         <y>10</y>
         <width>51</width>
         <height>23</height>
        </rect>
       </property>
       <property name="whatsThis">
        <string/>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_saveSHD_file">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>10</y>
         <width>91</width>
         <height>23</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>Open Shader File</string>
       </property>
       <property name="text">
        <string>Save 'SHD'  File :</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_saveSHD_file">
       <property name="geometry">
        <rect>
         <x>100</x>
         <y>10</y>
         <width>51</width>
         <height>23</height>
        </rect>
       </property>
       <property name="whatsThis">
        <string/>
       </property>
       <property name="text">
        <string>Save</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_saveImage">
       <property name="geometry">
        <rect>
         <x>333</x>
         <y>10</y>
         <width>71</width>
         <height>23</height>
        </rect>
       </property>
       <property name="toolTip">
        <string>Open Shader File</string>
       </property>
       <property name="text">
        <string>Save Image :</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="widget_2" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>530</y>
        <width>471</width>
        <height>81</height>
       </rect>
      </property>
      <widget class="QPushButton" name="pushButton_stepSix_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>453</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Save 'PRD' File (in-progress)</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_cleanFile">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>453</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Clean File !!!</string>
       </property>
      </widget>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>pushButton_open_001</tabstop>
  <tabstop>comboBox_types_list</tabstop>
  <tabstop>pushButton_open_002</tabstop>
  <tabstop>tabWidget_Tabs</tabstop>
  <tabstop>pushButton_saveSHD_file</tabstop>
  <tabstop>pushButton_ImageSaveFile</tabstop>
  <tabstop>pushButton_cleanFile</tabstop>
  <tabstop>pushButton_stepSix_3</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
